### ISSUE TITLE
lxd-images import lxc ubuntu (old syntax) throws stacktrace

### ISSUE THREAD

**Issue by raharper** [lxc/lxd#1384]
Running lxd (0.23-0ubuntu3) on Xenial, used lxd-import like I used to and got the error below.
Dropping 'lxc' from the command worked fine.

```
% lxd-images import lxc ubuntu wily amd64 --alias ubuntu --alias ubuntu/wily
error: argument source: invalid choice: 'lxc' (choose from 'busybox', 'ubuntu')
usage: lxd-images import [-h] {busybox,ubuntu} ...

positional arguments:
  {busybox,ubuntu}
    busybox         Busybox image
    ubuntu          Ubuntu images

optional arguments:
  -h, --help        show this help message and exit
Exception ignored in: <bound method LXD.__del__ of <__main__.LXD object at 0x7fd1321cd0f0>>
Traceback (most recent call last):
  File "/usr/bin/lxd-images", line 104, in __del__
TypeError: 'NoneType' object is not callable
```

**Comment by mikemccracken** [comment]
This should be fixed by #1377

**Comment by tych0** [comment]
Two errors here, he's complaining about that `lxc` doesn't give you a nice error message, instead of the __del__ issue. I'll post a patch in a sec.
